src/EvaluateCommand.php

```diff
@@ -1126,4 +1126,21 @@ protected function evaluateAndAddPoints($passes, $scored_points, $total_points)
         }
         $this->total += $total_points;
     }
+
+    /**
+     * @param string $download_path
+     *
+     * @return bool
+     */
+    protected function isOrcaIntegrated(string $download_path): bool
+    {
+        if (file_exists($download_path . '/travis.yml')) {
+            $travis_yml_contents = file_get_contents($download_path . '/travis.yml');
+            if (strpos($travis_yml_contents, 'orca') !== false) {
```

Comment:
I would suggest a minor modification, @grasmash:
```suggestion
            if (strpos($travis_yml_contents, 'ORCA_SUT_NAME') !== false) {
```